### PR TITLE
feat: add build for x86 with ecc608

### PR DESCRIPTION
**Pull request by shawaj**
- add build for x86_64 with ecc608
- mips-unknown-linux-musl was duplicated
- add more info to readme
- minor typo fixes

### CODE REVIEW

**Comment by shawaj** [comment]
@madninja if/when this gets merged, would it be possible to cut a new release after the merge?

File: Makefile.toml [hunk 1]
```diff
@@ -4,6 +4,9 @@ FEATURES = "ecc608"
 BUILD_COMMAND = "cross"
 TAR = { source = "${CARGO_MAKE_RUST_TARGET_OS}", default_value = "linux", mapping = {"macos" = "gtar", "linux" = "tar" } }
 
+[env.x86_64-unknown-linux-gnu]
```
Comment (madninja) [inline Makefile.toml]: Rename to `x86_64-ecc-debian-gnu` to match the tpm one?

Comment (shawaj) [reply Makefile.toml]: @madninja yeah can do if you think that is better?

I was mainly following the naming convention mentioned [here in the latest release](https://github.com/helium/gateway-mfr-rs/releases/tag/v0.4.1)...

```
Removes the used security feature at the end of the binary filename. All target profiles with unknown in their name can be assumed to be ecc608 targets. The x86_64-tpm-debian-gnu, for example is a tpm target .
```

But can change to whatever of course :-)

Comment (madninja) [reply Makefile.toml]: Ah yes.. so then `x86_64-unknonw-debian`. I tripped more over the use of linux than the tpm/unknown part

**Review by madninja — APPROVED** [review]

File: Makefile.toml (hunk 1 above)
Comment (shawaj) [reply Makefile.toml]: @madninja ok - fixed and rebased

**Review by madninja — APPROVED** [review]

**Pull request merged**